src/utils/formatters.ts

```diff
@@ -0,0 +1,20 @@
+export const truncateChannel = (channel: string, pad: number = 9) => {
+  const regex = /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{34}$/;
+
+  if (channel.match(regex) || channel.length >= 28) {
```

Comment:
Why an `OR` condition? This means the channel can match the regex or if it fails that it can be longer than 28 characters.

Reply:
The `OR` condition exists for custom user channel labels. A user can change the name of a channel to something to their liking. Usually, when these are displayed, they appear larger than a regular address. Therefore it was required for these channel labels to be truncated more aggresively than regular addresses.